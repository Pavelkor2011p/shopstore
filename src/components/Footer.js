import React from 'react'

export default function Footer() {
  return (
    <footer className='footer'>
      <div className='footer__items conteiner'>
        <div className='footer__items-logo'>
          <div className="footer__items-logo_white"></div>
          <p className='footer__items-logo_comp'>Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области</p>
          <p className='footer__items-logo_action'>Подпишись на скидки и акции</p>
          <div className="header__secondline-input footer__items-logo_input">
            <input type="search" placeholder="Введите ваш E-mail" className="header__secondline-input_search" />
            <input type="button" className="header__secondline-input_button header__secondline-input_button01" />
          </div>
        </div>
        <div className='footer__items-menu'>
          <h3>Меню сайта:</h3>
          <ul>
            <li>О компании</li>
            <li>Доставка и оплата</li>
            <li>Возврат</li>
            <li>Контакты</li>
          </ul>
        </div>
        <div className='footer__items-category'>
        <h3>Категории:</h3>
          <ul>
            <li>Бытовая химия</li>
            <li>Косметика и гигиена</li>
            <li>Товары для дома</li>
            <li>Товары для детей и мам</li>
            <li>Посуда</li>
          </ul>
        </div>
        <div className='footer__items-price'>
          <h3>Скачать прайс-лист:</h3>
          <button className="header__secondline-button button">Прайс-лист <div className='download'></div></button>
          <p>Связь в мессенджерах:</p>
            <div className='footer__items-price_mess'>
              <div className='footer__items-price_wa'></div>
              <div className='footer__items-price_tg'></div>
            </div>
        </div>
        <div className='footer__items-contacts'>
          <h3>Контакты:</h3>
          <p>+7 (777) 490-00-91</p>
          <p>время работы: 9:00-20:00<br></br>Заказать звонок</p>
          <p>opt.sultan@mail.ru 
          На связи в любое время</p>
          <div className='footer__items-contacts_card'>
            <div className='footer__items-contacts_visa'></div>
            <div className='footer__items-contacts_mastercard'></div>
          </div>
        </div>

      </div>
    </footer>
  )
}
