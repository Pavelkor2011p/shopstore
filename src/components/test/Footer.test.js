import{render, screen} from '@testing-library/react';
import Footer from '../Footer';



describe('Footer render', () => { 
    it('should create Footer with empty footer', () => {
        const utils = render(<Footer />);
        expect(utils).toMatchSnapshot();
    });

    it('list render', () => {
        render(<Footer />);

        expect(screen.getAllByRole('list')).toMatchSnapshot();
    })
 });