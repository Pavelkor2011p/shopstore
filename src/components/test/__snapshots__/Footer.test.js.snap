// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render list render 1`] = `
Array [
  <ul>
    <li>
      О компании
    </li>
    <li>
      Доставка и оплата
    </li>
    <li>
      Возврат
    </li>
    <li>
      Контакты
    </li>
  </ul>,
  <ul>
    <li>
      Бытовая химия
    </li>
    <li>
      Косметика и гигиена
    </li>
    <li>
      Товары для дома
    </li>
    <li>
      Товары для детей и мам
    </li>
    <li>
      Посуда
    </li>
  </ul>,
]
`;

exports[`Footer render should create Footer with empty footer 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer"
      >
        <div
          class="footer__items conteiner"
        >
          <div
            class="footer__items-logo"
          >
            <div
              class="footer__items-logo_white"
            />
            <p
              class="footer__items-logo_comp"
            >
              Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области
            </p>
            <p
              class="footer__items-logo_action"
            >
              Подпишись на скидки и акции
            </p>
            <div
              class="header__secondline-input footer__items-logo_input"
            >
              <input
                class="header__secondline-input_search"
                placeholder="Введите ваш E-mail"
                type="search"
              />
              <input
                class="header__secondline-input_button header__secondline-input_button01"
                type="button"
              />
            </div>
          </div>
          <div
            class="footer__items-menu"
          >
            <h3>
              Меню сайта:
            </h3>
            <ul>
              <li>
                О компании
              </li>
              <li>
                Доставка и оплата
              </li>
              <li>
                Возврат
              </li>
              <li>
                Контакты
              </li>
            </ul>
          </div>
          <div
            class="footer__items-category"
          >
            <h3>
              Категории:
            </h3>
            <ul>
              <li>
                Бытовая химия
              </li>
              <li>
                Косметика и гигиена
              </li>
              <li>
                Товары для дома
              </li>
              <li>
                Товары для детей и мам
              </li>
              <li>
                Посуда
              </li>
            </ul>
          </div>
          <div
            class="footer__items-price"
          >
            <h3>
              Скачать прайс-лист:
            </h3>
            <button
              class="header__secondline-button button"
            >
              Прайс-лист 
              <div
                class="download"
              />
            </button>
            <p>
              Связь в мессенджерах:
            </p>
            <div
              class="footer__items-price_mess"
            >
              <div
                class="footer__items-price_wa"
              />
              <div
                class="footer__items-price_tg"
              />
            </div>
          </div>
          <div
            class="footer__items-contacts"
          >
            <h3>
              Контакты:
            </h3>
            <p>
              +7 (777) 490-00-91
            </p>
            <p>
              время работы: 9:00-20:00
              <br />
              Заказать звонок
            </p>
            <p>
              opt.sultan@mail.ru На связи в любое время
            </p>
            <div
              class="footer__items-contacts_card"
            >
              <div
                class="footer__items-contacts_visa"
              />
              <div
                class="footer__items-contacts_mastercard"
              />
            </div>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer"
    >
      <div
        class="footer__items conteiner"
      >
        <div
          class="footer__items-logo"
        >
          <div
            class="footer__items-logo_white"
          />
          <p
            class="footer__items-logo_comp"
          >
            Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области
          </p>
          <p
            class="footer__items-logo_action"
          >
            Подпишись на скидки и акции
          </p>
          <div
            class="header__secondline-input footer__items-logo_input"
          >
            <input
              class="header__secondline-input_search"
              placeholder="Введите ваш E-mail"
              type="search"
            />
            <input
              class="header__secondline-input_button header__secondline-input_button01"
              type="button"
            />
          </div>
        </div>
        <div
          class="footer__items-menu"
        >
          <h3>
            Меню сайта:
          </h3>
          <ul>
            <li>
              О компании
            </li>
            <li>
              Доставка и оплата
            </li>
            <li>
              Возврат
            </li>
            <li>
              Контакты
            </li>
          </ul>
        </div>
        <div
          class="footer__items-category"
        >
          <h3>
            Категории:
          </h3>
          <ul>
            <li>
              Бытовая химия
            </li>
            <li>
              Косметика и гигиена
            </li>
            <li>
              Товары для дома
            </li>
            <li>
              Товары для детей и мам
            </li>
            <li>
              Посуда
            </li>
          </ul>
        </div>
        <div
          class="footer__items-price"
        >
          <h3>
            Скачать прайс-лист:
          </h3>
          <button
            class="header__secondline-button button"
          >
            Прайс-лист 
            <div
              class="download"
            />
          </button>
          <p>
            Связь в мессенджерах:
          </p>
          <div
            class="footer__items-price_mess"
          >
            <div
              class="footer__items-price_wa"
            />
            <div
              class="footer__items-price_tg"
            />
          </div>
        </div>
        <div
          class="footer__items-contacts"
        >
          <h3>
            Контакты:
          </h3>
          <p>
            +7 (777) 490-00-91
          </p>
          <p>
            время работы: 9:00-20:00
            <br />
            Заказать звонок
          </p>
          <p>
            opt.sultan@mail.ru На связи в любое время
          </p>
          <div
            class="footer__items-contacts_card"
          >
            <div
              class="footer__items-contacts_visa"
            />
            <div
              class="footer__items-contacts_mastercard"
            />
          </div>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
